### ISSUE TITLE
[Question] Can this be used to just reduce world lag on a single server?

### ISSUE THREAD

**Issue by nikolqy** [Jaxkr/mammoth#29]
Can WorldQL just be used to move the load of a world on a Spigot instance to a separate sever? I get this is meant to make a single world massively multiplayer, but realistically nobody can render more than 100 players at once without massive lag, and the data transfer rates would be insane. Great option for 2b2t though. But, let's say you wanted to hold 1k people on a single Spigot instance, would using WorldQL make this more possible by moving the world chunk loading to a separate server, or is this meant to be used with multiple servers and bigger setups?

Thanks!

**Comment by razorrider7** [comment]
This allows entities and worlds to sync between servers. It does not move chunk loading off the main server.

Each server starts with the same world, and loads the chunks regularly. If anything makes changes to the world, the changes get saved to the database and sent to the other servers to copy.

For example, if you load up a new server and sync it with the database, it will automatically make all the changes listed in the database. It will then be up to date with the other servers.

TLDR: WorldQL just syncs changes between servers, it does not move chunk loading to another server.

**Comment by Jaxkr** [comment]
@razorrider7 is correct. Mammoth will not improve performance for a single server. However, cross server PVP for players close together does work properly. We hope to enable PVP tournaments with 100s of players, the client doesn't struggle too badly with it in our tests.
